tensorflow/contrib/learn/python/learn/estimators/head.py

```diff
@@ -37,7 +37,8 @@
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import control_flow_ops
-from tensorflow.python.ops import logging_ops
```

Comment:
I tried to use assertEqual but 46 tests are failing. head_test.py is outdated. Would you like to give some suggestions?